hs_connectors/Makefile

```
@@ -0,0 +1,6 @@
+CHECKDIRS := src setup.py
+
+# clean package
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)